drivers/bridge/bridge.go

```diff
@@ -380,13 +380,6 @@ func (d *driver) configure(option map[string]interface{}) error {
 		return &ErrInvalidDriverConfig{}
 	}
 
-	if config.EnableIPForwarding {
```

Comment:
Given the new logic is to change the FORWARD chain default policy only if `config.EnableIPTables==true`, I am thinking we do not need to move this code, neither make any change in `SetupForwarding()`.
We only need to add in `SetupIPChain(configuration)`

```
if config.EnableIPForwarding {
    if err := iptables.SetDefaultPolicy(iptables.Filter, "FORWARD", iptables.Drop); err != nil {
        return fmt.Errorf("%v. IP forwarding was set to 1", err)
    }
}
```

Also this will guarantee the action will take place when the firewall is reloaded: 
`iptables.OnReloaded(..., setupIPChains)`